src/services/billing/cost-management/budget/budget-detail/modules/budget-notifications/BudgetNotificationsChannel.vue

```diff
@@ -41,22 +49,42 @@ import { ApiQueryHelper } from '@spaceone/console-core-lib/space-connector/helpe
 import { SpaceConnector } from '@spaceone/console-core-lib/space-connector';
 import ErrorHandler from '@/common/composables/error/errorHandler';
 
+const getBadgeColor = (level: string) => {
+    switch (level) {
+    case 'LV1':
+        return 'secondary1';
+    case 'LV2':
+        return 'indigo';
+    case 'LV3':
+        return 'peacock';
+    case 'LV4':
+        return 'coral500';
+    case 'LV5':
+        return 'alert';
+    default:
+        return 'primary';
+    }
+};
+
```

Comment:
단순하게 컬러 스트링만 리턴하는거라면, object map을 이용하면 어떨까요?

```suggestion
const BADGET_COLOR_MAP: Record<string, string> = {
    LV1 : 'secondary1',
    LV2:  'indigo',
    LV3: 'peacock',
    LV4: 'coral500',
    LV5: 'alert',
};
```

Reply:
default color를 따로 매핑해줘야 해서 기존 방식이 한 쪽에서 몰아볼 수 있어서 더 좋은 것 같기도 하고요. 😮

Reply:
성능면에서 이렇게하는게 더 좋다고 해서 제안드려봤지만, 가독성과 효율성이 더 떨어져보입니다. ㅠㅠ
제 제안은 무시하셔도 되어요 ㅎㅎ

Reply:
오 저렇게 써보고싶은데 제가 사실 아직 저렇게 매핑하는거에 익숙하지않아서 잘 생각이 안나더라구여 ㅎㅎㅎ 써볼까요?!?

Reply:
default 케이스를 저렇게 따로 해주는게 더 번거로울것 같기도 해서 ㅠㅠ 편한 방식으로 해주세요!!